### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by kris-ellery** [LeaVerou/conic-gradient#8]
Any thoughts on implementing mobile support? Currently it is not working and I couldn't find any reference to mobile support in documentation.

**iPhone 6, iOS 8.3, Safari**

![img_0468](https://cloud.githubusercontent.com/assets/2780894/8340983/a7216954-1a87-11e5-897a-f61364133edb.png)

**Comment by LeaVerou** [comment]
The issue is with Safari in general, not just mobile. 
It's probably easy to fix and I will look into it at some point soon, but I was hoping someone else will debug it earlier and send a PR :P

**Comment by kris-ellery** [comment]
Ahh, makes sense. Must be something with WebKit as Chrome on iOS doesn't work as well.

**Comment by ganlanyuan** [comment]
Seems IE 9 ~ 11 don't work as well.
![ie11](https://cloud.githubusercontent.com/assets/4998811/9707790/e7b6848a-54df-11e5-9a1b-755a47553309.png)

**Comment by eonj** [comment]
Current version of this polyfill depends on at least two browser feature: 2D drawing context and `toDataURL()` of HTML5 Canvas, and SVG image support with `<image>` tag provided via data URI. IE 8~11 and EdgeHTML 14.14393 also lacks the latter.

**Comment by kumarharsh** [comment]
@Uni- I think EdgeHTML **does** support it...

http://codepen.io/chriscoyier/full/ZQgvyG/

in Edge Mobile (W10M EdgeHTML 15.14965)
![wp_ss_20161223_0001](https://cloud.githubusercontent.com/assets/760112/21450867/7ea11666-c920-11e6-9823-9680470f19a4.png)

**Comment by eonj** [comment]
@kumarharsh You're right. Base64 data URI image is supported, but I think there is bug on using it background image.

Edge (EdgeHTML 14.14393):

![Edge](https://cloud.githubusercontent.com/assets/1371115/21450991/05f24928-c93f-11e6-81b4-8d0d96ea1796.png)

IE 11 (11.576.14393):

![IE 11](https://cloud.githubusercontent.com/assets/1371115/21450997/0cfce322-c93f-11e6-9a8f-9594d02e2471.png)

**Comment by kumarharsh** [comment]
@Uni- oh yes, you're right. I didn't notice that. Let's hope something improves with Edge soon...
Update: Filed a bug report: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/10304656/

**Comment by kumarharsh** [comment]
@Uni- I've verifiable information that Edge **does** support the whole spec. The only reason those examples don't render is because Edge wants everything after the `utf8` part to be escaped. This is a platform difference between IE/Edge and other browsers. Use `escape(SVG_URI)` to verify.

Ref: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/7157459/

Reference image in Edge:

![image](https://cloud.githubusercontent.com/assets/760112/22920644/2be22058-f2bc-11e6-8141-a48c8f0f8c31.png)

And here's a PR for it: https://github.com/LeaVerou/conic-gradient/pull/21

**Comment by kumarharsh** [comment]
@KrisOlszewski @LeaVerou it's working for Edge mobile too

![wp_ss_20170214_0001 1](https://cloud.githubusercontent.com/assets/760112/22930786/06342d0c-f2e8-11e6-83da-ad039a54e51c.png)

**Comment by forresto** [comment]
Works from iOS v9...
<img width="1181" alt="image" src="https://user-images.githubusercontent.com/395307/53642412-bae85b80-3bff-11e9-8710-223e69b4a402.png">
